modin/experimental/core/execution/native/implementations/omnisci_on_native/dataframe/dataframe.py

```diff
@@ -333,14 +333,18 @@ def take_2d_labels_or_positional(
                 force_execution_mode=self._force_execution_mode,
             )
 
-        if row_labels is not None or row_positions is not None:
+        if row_labels is not None:
+            raise NotImplementedError("row indices masking is not yet supported")
```

Comment:
```suggestion
            raise NotImplementedError("Row labels masking is not yet supported")
```

Reply:
Do we get an exception in the tests?

Reply:
> Do we get an exception in the tests?
Sorry, I didn't get the question.

Reply:
As of now we raise the exception here. Do we have a test case for this?

Reply:
No, there is no a test case since this functionality is not currently supported.